src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -45,11 +44,8 @@ internal interface IEslintBridgeProcess : IDisposable
         void Stop();
     }
 
-    [Export(typeof(IEslintBridgeProcess))]
-    [PartCreationPolicy(CreationPolicy.Shared)]
     internal sealed class EslintBridgeProcess : IEslintBridgeProcess
```

Comment:
EslintBridgeProcess is no longer singleton, and it has a factory that is responsible for importing `EslintBridgeDirectoryMefContractName`